www/grpc/proto/network.proto

```diff
@@ -129,3 +131,13 @@ message CounterInfo {
   // Total number of bundles.
   uint64 Bundles = 2;
 }
+
+// FeeConfig holds the fee configuration that stored in node config.
+message FeeConfig {
+  // Fixed fee for each transaction.
+  double fixed_fee = 1;
+  // Number of bytes an account can send each day without paying a fee.
```

Comment:
Please add this comment here: `The `daily_limit` is part of he consumptional fee model.`